app/notify_client/notification_api_client.py

```diff
@@ -127,5 +127,8 @@ def get_notification_status_by_service(self, start_date, end_date):
             }
         )
 
+    def get_notification_count_for_job_id(self, *, service_id, job_id):
```

Comment:
Don’t you need the `*` here? I can’t see where you’re using it.
```suggestion
    def get_notification_count_for_job_id(self, service_id, job_id):
```

Reply:
The star makes it necessary to name the arguments following it when calling function. It's a style choice, I like named arguments, they are readable and leave less place for mistakes.

Reply:
Cool, I learned something! Makes a lot sense when you’re passing in two UUIDs.

Reply:
Yay, I learnt on Pycon UK about the star :)